vars/commonPipelineEnvironment.groovy

```diff
@@ -176,16 +177,16 @@ class commonPipelineEnvironment implements Serializable {
         containerProperties.each({key, value ->
             def fileName = ".pipeline/commonPipelineEnvironment/container/${key}"
             if (value && !script.fileExists(fileName)) {
-                //ToDo: check for value type and act accordingly?
-                script.writeFile file: fileName, text: value
+                // Format data in case value is complex type
```

Comment:
@stiller-leser we should ensure that Strings are going to be serialized plainly and everything else as a JSON. This is basically to keep backward compatibility.